lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -132,7 +137,8 @@ class AndroidNotificationDetails {
       this.ledColor,
       this.ledOnMs,
       this.ledOffMs,
-      this.ticker});
+      this.ticker,
+      this.scheduledAndroidNotificationPrecision = ScheduledAndroidNotificationPrecision.Inexact});
```

Comment:
Do you think it might be better to set the default as Exact for a minor version increment, then change it to Inexact for a major increment?

Reply:
Should be the other way around if exact was the default as that's a breaking change in behaviour and thus should be a major version change. I'd be inclined to leave it as inexact unless all the methods for setting up recurring notifications can use the new enum. Reason being is that without that change at least the Android behaviour is consistent. If the default was to be so that it's consistent with iOS then if I'm not mistaken, it should be exact and allowed while idle

Reply:
Hmm just realised I may have confused myself and as a result caused some confusion whilst looking at this PR, so apologies first about that. If you look at the original code I had it actually was for exact timing. If the call to `set` was done in API version > 19 (i.e. beyond KitKat) then it would've been inexact. However, there were API version checks that make sure it calls exact. The implementation of `AlarmManagerCompat.setExact` does the same thing as the code I originally had. So you're right in that the default behaviour as it stands right now is exact. Though this means the default value should also be set to exact if backwards compatibility is to be maintained

Reply:
Following on from that point, I think we could remove the inexact option as it calls `AlarmManager.set`, which will be exact on older Android versions